nginx/files/_log.conf

```diff
@@ -1,3 +1,5 @@
+{%- from "nginx/map.jinja" import server with context -%}
+{%- set site = salt['pillar.get']('nginx:server:site:'+site_name) %}
 
-  access_log /var/log/nginx/{{ site_name }}-access.log;
-  error_log /var/log/nginx/{{ site_name }}-error.log;
+  access_log {{ site.get('access_log', server.log_dir + '/' + site_name + '-access.log') }};
```

Comment:
I would prefer this structure of metadata:

```
site:
  log:
    access: 
    error: 
```

Reply:
Ok, I can change that.

And for the main server logs, should I use the same metadata structure?

Reply:
Yes, that would be great

Reply:
The structure of log metadata is now updated with commit: d63beda